scss/bootstrap/overrides/style/_ui-select.scss

```diff
@@ -1,7 +1,5 @@
-.ui-select-bootstrap[theme="civihr-ui-select"] {
-
+.ui-select-bootstrap[theme='civihr-ui-select'] {
```

Comment:
this is a bootstrap/ partial, what does it have to do with core screens?

Reply:
This is stylelint change, nothing to do with core screen.